cohort-cli/pom.xml

```diff
@@ -1,3 +1,8 @@
+<!--  ************************************** -->
+<!--  (C) Copyright IBM Corp. 2020, 2020     -->
```

Comment:
I know you mentioned something about copyright cleanup so if making these all say 2021 later is an item I'll let them go.

Reply:
Cleaned up the ones in these files, at least